pkg/oc/admin/prune/imageprune/prune.go

```diff
@@ -662,6 +668,25 @@ func (p *pruner) addBuildsToGraph(builds *buildapi.BuildList) []error {
 	return errs
 }
 
+// resolveISTagName parses  and tries to find it in the graph. If the parsing fails,
+// an error is returned. If the istag cannot be found, nil is returned.
+func (p *pruner) resolveISTagName(g genericgraph.Graph, referrer *kapi.ObjectReference, istagName string) (*imagegraph.ImageStreamTagNode, error) {
+	name, tag, err := imageapi.ParseImageStreamTagName(istagName)
+	if err != nil {
+		if p.ignoreInvalidRefs {
+			glog.Warningf("Failed to parse ImageStreamTag name %q: %v", istagName, err)
+			return nil, nil
+		}
+		return nil, newErrBadReferenceTo("ImageStreamTag", istagName, referrer, err.Error())
+	}
+	node := g.Find(imagegraph.ImageStreamTagNodeName(makeISTag(referrer.Namespace, name, tag)))
+	if istNode, ok := node.(*imagegraph.ImageStreamTagNode); ok {
+		return istNode, nil
+	}
+
+	return nil, nil
```

Comment:
would it be worth having an "isNotFound" error or something similar that a caller of this method could check?

Reply:
@juanvallejo We return `nil` if nothing is found. It's not an error for this function. Do not need to return an error where it does not exist. The caller function  already handles the `istNode == nil`.